### PR TITLE
mes-4111 Check and update Category A Mod1 waiting room screen

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Mes-4111 Created the CBT number component for the category A mod1 waiting room screen.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2020-01-23 at 10 40 44

### CODE REVIEW

File: src/modules/tests/pre-test-declarations/cat-a-mod1/pre-test-declarations.cat-a-mod1.selector.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { PreTestDeclarations } from '@dvsa/mes-test-schema/categories/AM1/index';
+
+export const getcbtNumberStatus = (decs: PreTestDeclarations) => decs.DL196CBTCertNumber;
```
Comment (MarkPatt89) [inline src/modules/tests/pre-test-declarations/cat-a-mod1/pre-test-declarations.cat-a-mod1.selector.ts]: I'd be inclined to change this to camelCase: `getCBTNumberStatus`. Your call though.

File: src/pages/waiting-room/components/cbt-number/__tests__/cbt-number.spec.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    describe('cbtNumberChanged', () => {
+      it('It should emit a cbt number if the characters only contain numbers', () => {
+        spyOn(component.cbtNumberChange, 'emit');
+        const cbtNumber = '12345678';
+        component.cbtNumberChanged(cbtNumber);
+        expect(component.cbtNumberChange.emit).toHaveBeenCalledWith(cbtNumber);
+      });
+    });
+
+    describe('invalid', () => {
```
Comment (MarkPatt89) [inline src/pages/waiting-room/components/cbt-number/__tests__/cbt-number.spec.ts:41]: this `describe` message doesn't make much sense.

Comment (MarkPatt89) [reply src/pages/waiting-room/components/cbt-number/__tests__/cbt-number.spec.ts:41]: Wait it does. It's just testing the `invalid()` function in the `ts` file, yeah?

**Review by MarkPatt89 — APPROVED** [review]
Looks good

**Review by TomBillingtonUK — APPROVED** [review]

File: src/pages/waiting-room/components/cbt-number/__tests__/cbt-number.spec.ts (hunk 2 above)
Comment (appleabs) [reply src/pages/waiting-room/components/cbt-number/__tests__/cbt-number.spec.ts:41]: yea :)

**Pull request merged**